package.json

```diff
@@ -122,8 +122,8 @@
     "ca-gen": "./bin/update-ca-bundle.sh",
     "docker-env": "./bin/docker-env-vars.sh",
     "docs": "npm ci && jsdoc -c ./jsdoc-conf.json --private -r .",
-    "integration": "npm run prepare-test && npm run sub-install && time tap test/integration/**/**/*.tap.js --timeout=180 --no-coverage",
-    "prepare-test": "npm run ca-gen && npm run ssl && npm run docker-env",
+    "integration": "npm run prepare-test && npm run sub-install && time tap test/integration/**/**/*.tap.js --timeout=180 --no-coverage --reporter classic",
+    "prepare-test": "npm ci && npm run ca-gen && npm run ssl && npm run docker-env",
```

Comment:
Do we need `npm ci` here in prepare-test?

Reply:
I don't think so. I removed this a few weeks ago because in CI we run npm install before calling any other scripts

Reply:
i will fix this and then approve